doc-content/jbpm-docs/src/main/asciidoc/Overview/processengine-overview-con.adoc

```diff
@@ -0,0 +1,45 @@
+[id='processengine-overview-con']
+= Overview of the {PROCESS_ENGINE}
+
+The {PROCESS_ENGINE} implements the Business Process Management (BPM) paradigm. BPM is a business methodology that enables modeling, measuring, and optimizing processes within an enterprise. 
+
+In BPM, a repeatable business process is represented as a workflow diagram. The Business Process Management Notation (BPMN) specification defines the available elements of this diagram. The {PROCESS_ENGINE} implements a large subset of the BPMN 2.0 specification.
+
+With {PROCESS_ENGINE}, business analysts can develop the diagram itself. Developers can implement the business logic and, when necessary, user interface of every element of the flow in code, making an executable business process. Users can execute the business process and interact with it as necessary. Then analysts can generate metrics that reflect the efficiency of the process.
+
+The workflow diagram consists of a number of nodes. The BPMN specification defined many kinds of nodes. The principal types of node are:
+
+* _Event_: nodes representing something happening outside of the process. Typical events are the start and the end of a process. An event can throw messages to other processes and catch such messages. Circles on the diagram represent events.
+* _Activity_: nodes representing an action that must be taken (whether automatically or with user involvement). Typical events are a task, which represents an action taken within the process, and a call to a subprocess. Rectangles on the diagram represent activities.
+* _Gateway_: a branching node. A typical gateway evaluates an expression and, depending on the result, continues to one of several execution paths. Diamond shapes on the diagram represent gateways.
+
+When a user starts the process, a process instance is created. The process instance contains a set of data, or _context_, stored in process variables. 
+
+Some of these variables can be initialized when a user starts the process. An activity can read from process variables and write to process variables. A gateway can evaluate process variables to determine the execution path.
+
+For example, a purchase process in a shop can be a business process. The content of the user’s cart can be the initial process context. At the end of execution, the process context can contain the payment confirmation and shipment tracking details. 
+
+Optionally, you can use the BPMN data modeler in {CENTRAL} to design the model for the data in process variables.
+
+The workflow diagram is represented in code by an XML _business process definition_. The logic of events, gateways, and subprocess calls are defined within the business process definition. However, tasks are not implemented within the definition. Instead, for every task activity, the {PROCESS_ENGINE} executes a call using the _Work Item Handler API_. A number of tools can implement this API, providing a flexible mechanism for implementing various tasks.
```

Comment:
Maybe replace "tasks are not implemented within the definition ... " with "For custom tasks, the actual implementation logic is externalized as integration code, using the _Work Item Handler API_"